.travis.yml

```diff
@@ -2,12 +2,15 @@ os: osx
 language: objective-c
 osx_image: xcode9
 
+install:
+  - brew update && brew install swiftlint 
+
 before_script:
+  - swiftlint
```

Comment:
I'm not sure if this will have any effect for warning violations rules. What about removing in favor of `Danger` integration once implemented?

Reply:
Yeah, that was the plan indeed :) 
I just wanted something until we have Danger working and didn't want to have everything in a single PR.